pkg/rest/echo_interface.go

```diff
@@ -15,7 +15,7 @@ type echoRouter interface {
 }
 
 type echoServer interface {
-	echoRouter
```

Comment:
This is not needed anymore since [1ed4447](https://github.com/Knoblauchpilze/user-service/commit/1ed44472ecbbca766ce3954e9d71baf8fca64ae9#diff-b94bcc70262942e92621180995672624820602b1042b2f62036a4acf2a150b6cL87) as we now use the public and authorized routers to create the routes and not the server anymore.